lib/main.js

```diff
@@ -5,10 +5,25 @@ var parse = require("./parse.js");
 var filters = {};
 var Lexer = parse.Lexer;
 var Parser = parse.Parser;
+var parserOptions = {
+    csp: false, // noUnsafeEval,
```

Comment:
Ah ok, now I get it. It tells the parser whether it's allowed to use `eval()` in order to compile the expression. When `csp` is `true`, angular can't use this optimization because it's forbidden by the CSP.